scripts/table_diff/changes/testdata/pr_6012_diff.txt

```diff
@@ -0,0 +1,1042 @@
+diff --git a/plugins/source/github/codegen/recipes/actions.go b/plugins/source/github/codegen/recipes/actions.go
```

Comment:
We can keep only the diff related to the `.md` files to make it easier to understand this file